flare/dft_interface/vasp_util.py

```diff
@@ -0,0 +1,87 @@
+from pymatgen.io.vasp.inputs import Poscar
+from pymatgen.io.vasp.outputs import Vasprun
+from subprocess import call
+import numpy as np
+from flare import struc
+from typing import List
+from json import dump, load
+
+
+def run_dft_command(calc_dir: str, dft_loc: str,
+	structure: struc.Structure=None, en: bool=False):
+	
+	currdir = os.getcwd()
+	if structure:
+		edit_dft_input_configurations("POSCAR", structure)
+	try:
+		os.chdir(calc_dir)
+	except Exception, e:
+		os.chdir(currdir)
+		call(vasp_cmd, shell=True)
+		raise e
```

Comment:
Perhaps could change this to an if statement to see if calc dir and calc dir disagree, and then if so, use the chdir command to change to calc dir?